README.md

```diff
@@ -49,6 +59,57 @@ Then:
 * If you chose to use Jenkins it depends on your internal Jenkins setup.
   * Required Jenkins plugins: `Warnings Next Generation Plugin >=4.0.0`, `Cobertura Plugin >=1.13`,  `Pipeline: GitHub Groovy Libraries >=1.0`
   * For internal TomTom projects: Trigger **Scan Organization Folder Now** on the BitBucket Team Project folder. In case of problems contact the @nav-pipeline team.
+* Contact the @nav-pipeline team to activate your project on `pyup.io`.
```

Comment:
Do you still want to mention it giving the recent addition of Dependabot?

Reply:
Hm.. I actually removed it here: https://github.com/tomtom-international/cookiecutter-python/pull/42/files#diff-04c6e90faac2675aa89e2176d2eec7d8L40

Probably wrongly merged

Reply:
Oh, I missed that place. There were two locations